src/benchmark.cc

```diff
@@ -131,50 +142,41 @@ bool CpuScalingEnabled() {
   return false;
 }
 
-// Given a collection of reports, computes their mean and stddev.
-// REQUIRES: all runs in "reports" must be from the same benchmark.
 void ComputeStats(const std::vector<BenchmarkReporter::Run>& reports,
                   BenchmarkReporter::Run* mean_data,
                   BenchmarkReporter::Run* stddev_data) {
   // Accumulators.
   Stat1_d real_accumulated_time_stat;
   Stat1_d cpu_accumulated_time_stat;
-  Stat1_d items_per_second_stat;
   Stat1_d bytes_per_second_stat;
-  Stat1_d iterations_stat;
-  Stat1MinMax_d max_heapbytes_used_stat;
+  Stat1_d items_per_second_stat;
+  int64_t total_iters = 0;
 
   // Populate the accumulators.
   for (std::vector<BenchmarkReporter::Run>::const_iterator it = reports.begin();
-       it != reports.end(); ++it) {
+       it != reports.end();
+       it++) {
     CHECK_EQ(reports[0].benchmark_name, it->benchmark_name);
+    total_iters += it->iterations;
```

Comment:
if you're not iteration bound then multiple runs (repetitions) may have different iteration counts. when it comes to printing the mean column, it's weird to print the total iterations in that column. the fixed version currently implemented tracks it as a stat so it is correctly reported.